.kitchen.yml

```diff
@@ -134,17 +134,55 @@ suites:
       - fedora-30
       - opensuse-15
       - opensuse-42
-  - name: py2-stable-2019-2
+  - name: py3-git-2018-3
+    provisioner:
+      salt_version: 2018.3
+      salt_bootstrap_options: -x python3 -MPfq git %s
+    excludes:
+      - amazon-1
+      - centos-6
+      - debian-8
+      - fedora-29
+      - fedora-30
+      - opensuse-15
+      - opensuse-42
+  - name: py3-git-2019-2
     provisioner:
       salt_version: 2019.2
-      salt_bootstrap_options: -MP stable %s
+      salt_bootstrap_options: -x python3 -MPfq git %s
```

Comment:
why quiet ?

Reply:
i was using the same opts that we are currently using for git installs: https://github.com/bryceml/salt-bootstrap/blob/6b24948a911938dcb4fac73f9d278104175a19a4/.kitchen.yml#L17

its also nice because you dont see the output from all of the filepath names when installing with the bootstrap with a git install.

Reply:
oh, I see, the git diff just was weird since it changed from stable to git.